### PR TITLE
Athena-312 Added error message in case of unavailable vocabulary

### PR DESCRIPTION
**Pull request by johnSamilin**

### CODE REVIEW

File: src/modules/Vocabulary/components/DownloadHistory/components/ModalRequestLicenses/container.ts [hunk 1]
```diff
@@ -0,0 +1,81 @@
+/*
+ *
+ * Copyright 2018 Observational Health Data Sciences and Informatics
```
Comment (mpozhidaeva) [inline src/modules/Vocabulary/components/DownloadHistory/components/ModalRequestLicenses/container.ts]: new

File: src/modules/Vocabulary/components/DownloadHistory/components/ModalRequestLicenses/presenter.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    licenses,
+    goToLicenses,
+  } = props;
+  const classes = BEMHelper('modal-licenses');
+
+  return (
+    <div {...classes()}>
+      <Modal modal={modal} title='Licenses needed'>
+        <p {...classes('message')}>
+          You need some licenses ({licenses.length}) in order to download this bundle. Please, request them on Download page. The required vocabularies will be pre-selected.
```
Comment (mpozhidaeva) [inline src/modules/Vocabulary/components/DownloadHistory/components/ModalRequestLicenses/presenter.tsx]: for restore?

File: src/modules/Vocabulary/components/DownloadHistory/components/ModalRequestLicenses/container.ts (hunk 1 above)
Comment (johnSamilin) [reply src/modules/Vocabulary/components/DownloadHistory/components/ModalRequestLicenses/container.ts]: what?

Comment (agackovka) [reply src/modules/Vocabulary/components/DownloadHistory/components/ModalRequestLicenses/container.ts]: * Copyright 2018 Odysseus Data Services, inc.

**Pull request merged**